clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -554,4 +554,51 @@ public String getUrl() {
     // Return the default localhost URL
     return DEFAULT_URL;
   }
+
+  /* Methods used for testing - basically a seam where we can see what it passed and use mocks to see if the correct command is created. */
+
+  protected MetalakeAudit createMetalakeAudit(String url, boolean ignore, String metalake) {
+    return new MetalakeAudit(url, ignore, metalake);
+  }
+
+  protected MetalakeDetails createMetalakeDetails(String url, boolean ignore, String metalake) {
+    return new MetalakeDetails(url, ignore, metalake);
+  }
+
+  protected ListMetalakes createListMetalakes(String url, boolean ignore) {
+    return new ListMetalakes(url, ignore);
+  }
+
+  CreateMetalake createCreateMetalake(String url, boolean ignore, String metalake, String comment) {
```

Comment:
Function name is a little bit awkward ... 
Maybe change these methods to "newXYZ"?

Reply:
Thanks for the suggestion.